templates/download/risc-v/index.html

```diff
@@ -57,6 +58,7 @@ <h1 class="p-heading--2"><strong>Download Ubuntu for RISC-V Platforms</strong></
       {% include "download/risc-v/tab-4.html" %}
       {% include "download/risc-v/tab-5.html" %}
       {% include "download/risc-v/tab-6.html" %}
+      {% include "download/risc-v/tab-7.html" %}
```

Comment:
I think this is meant to be 3rd tab from the top (under StarFive VisionFive). There is a comment in copydoc that seems to suggest this.